etc/kayobe/kolla.yml

```diff
@@ -352,18 +352,22 @@ kolla_build_blocks:
     {% if stackhpc_kolla_clean_up_repo_mirrors | bool %}
     {% if kolla_base_distro in ['centos', 'rocky'] %}
     RUN \
-        tar -xzf /etc/yum.repos.d.backup/repos.tar.gz -C /etc/yum.repos.d && \
+        if [ -f /etc/yum.repos.d.backup/repos.tar.gz ]; then \
+          tar -xzf /etc/yum.repos.d.backup/repos.tar.gz -C /etc/yum.repos.d
+        fi && \
         if grep -r '{{ stackhpc_repo_mirror_url }}' /etc/yum.repos.d; then \
-           echo "Found repository mirror in Yum repositories"; \
-           exit 1; \
+          echo "Found repository mirror in Yum repositories"; \
+          exit 1; \
         fi && \
         rm -rf /etc/yum.repos.d.backup/
```

Comment:
probably still need this is the `if [ -f /etc/yum.repos.d.backup/repos.tar.gz ]; then \` block

Reply:
I think you meant about the ``rm rf /etc/yum.repos.d.backup/`` here? I've moved it into the block, although given ``rm`` still runs even if the file doesn't exist, it probably would have been fine :wink: